utils/ansible_galaxy_meta_template.yml

```diff
@@ -1,5 +1,5 @@
 galaxy_info:
-  author: SCAP Security Guide development team
+  author: Red Hat Security Automation development team
```

Comment:
ComplianceAsCode has a lot of contributors outside Red Hat.  I think we can't say that the author of the roles is only a Red Hat team. I prefer to put here "ComplianceAsCode contributors" or "ComplianceAsCode development team"

Reply:
Good point! Fixed @jan-cerny